### ISSUE TITLE
Make presets available via icon-buttons

### ISSUE THREAD

**Issue by almarklein** [almarklein/timetagger#445]
A bit like [this](https://play.google.com/store/apps/details?id=com.razeeman.util.simpletimetracker)
<img src='https://github.com/almarklein/timetagger/assets/3015475/ee5ac1b0-31d7-4e8c-8664-ee25b0a75291' width=250 />

**Comment by drajabr** [comment]
OMG this is freakin awesome!

**Comment by smibrandon** [comment]
If someone could explain the API PUT request to start/stop a description event, I could easily provide instructions and build the backbone to put this into MacroDroid and Home Assistant

**Comment by almarklein** [comment]
> If someone could explain the API PUT request to start/stop a description event

Can you be more clear what part is unclear to you? Perhaps these resources help?

* The web api docs: https://timetagger.readthedocs.io/en/latest/webapi/
* How the timetagger cli uses the API: https://github.com/almarklein/timetagger_cli/blob/main/timetagger_cli/core.py
  * Of particular interest are `start()`, `stop()` and  `request()`.

**Comment by kwillno** [comment]
Hi, I have made (something similar to) this using Shortcuts in iOS. If anyone would like to copy those just tag me. 

My experience from setting that up is that working with the API requires a lot local processing of  data between requests. Is there any way to simplify working with the API @almarklein? Akin to sending a request with only the string to set for the timer and then have the server figure out all the timing?

**Comment by almarklein** [comment]
> Is there any way to simplify working with the API

No, the server is pretty "stupid", as it knows little about what the data means.

**Comment by GinSoakedBoy** [comment]
> Hi, I have made (something similar to) this using Shortcuts in iOS. If anyone would like to copy those just tag me.
> 
> My experience from setting that up is that working with the API requires a lot local processing of data between requests. Is there any way to simplify working with the API [@almarklein](https://github.com/almarklein)? Akin to sending a request with only the string to set for the timer and then have the server figure out all the timing?

@kwillno, I know this is an old thread, but if you still have your iOS shortcuts for Timetagger, I'd love to take a look.
